.github/workflows/commit-preview.yaml

```diff
@@ -51,6 +51,7 @@ jobs:
           retention-days: 1
 
       - name: Publish to Pages
+        if: github.repository == 'QuiltMC/quiltmc.org'
```

Comment:
shouldn't it check for existence of the token instead ?

Reply:
Secrets cannot be referenced directly, and I did not want to first store it in an environment variable or previous step output.
see https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-using-secrets

Reply:
What's the issue with storing it in an environment variable ? It seems like the recommended way

Reply:
I guess if we did that we should ideally split the last two steps to a separate job though

Reply:
Possibly, yeah.